code/modules/clothing/suits/armor.dm

```diff
@@ -78,17 +78,31 @@
 
 /obj/item/clothing/suit/armor/riot
 	name = "riot suit"
-	desc = "A suit of armor with heavy padding to protect against melee attacks. Looks like it might impair movement."
+	desc = "A suit of armor with heavy padding to protect against melee attacks."
 	icon_state = "riot"
 	item_state = "swat_suit"
 	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
 	cold_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
 	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
-	armor = list(melee = 50, bullet = 10, laser = 10, energy = 10, bomb = 0, bio = 0, rad = 0)
+	armor = list(melee = 70, bullet = 10, laser = 10, energy = 10, bomb = 0, bio = 0, rad = 0)
 	flags_inv = HIDEJUMPSUIT
 	strip_delay = 80
 	put_on_delay = 60
 
+/obj/item/clothing/suit/armor/eod
+	name = "EOD suit"
```

Comment:
Call it "Bulldozer" armor, I DARE you. Realisticlly though, do not called it "EOD" (Explosive Ordinance Disposal). I do not think it should have full bomb protections, either. You already have bomb (EOD) suits in tasteful security colors. It would be confusing to have this as the bomb suit.

__Normal bomb suits have a higher slowdown(2) and were nerfed to 20 melee armor.__ This would make this suit a straight upgrade, making the ACTUAL EOD suit obsolete.